Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Location/UpdateLocation.hs

```diff
@@ -143,30 +143,34 @@ updateLocationHandler UpdateLocationHandle {..} waypoints = withLogTag "driverLo
     unless (driver.role == Person.DRIVER) $ throwError AccessDenied
     LocUpd.whenWithLocationUpdatesLock driver.id $ do
       mbOldLoc <- findDriverLocation
-      case filterNewWaypoints mbOldLoc minLocationAccuracy of
-        [] -> logWarning "Incoming points are older than current one, ignoring"
+      let sortedWaypoint = toList $ NE.sortWith (.ts) waypoints
+          filteredWaypoint = maybe sortedWaypoint (\oldLoc -> filter ((oldLoc.coordinatesCalculatedAt <) . (.ts)) sortedWaypoint) mbOldLoc
+      mbRideIdAndStatus <- getAssignedRide
+
+      case filteredWaypoint of
+        [] -> pure ()
```

Comment:
Please add a logwarning that says "Incoming points are older than current one, or points have bad accuracy, ignoring".

Reply:
why? here we didn't check accuracy yet

Reply:
Updated the text.